skiko/src/commonTest/kotlin/org/jetbrains/skia/ManagedStringTest.kt

```diff
@@ -21,4 +22,37 @@ class ManagedStringTest {
         }
         assertEquals("你好", s3)
     }
+
+    @Test
+    fun canCreateAndReadManagedString() {
+        val ms1 = ManagedString("Hello")
+        assertEquals("Hello", ms1.toString())
+
+        val ms2 = ManagedString("Привет!")
+        assertEquals("Привет!", ms2.toString())
+
+        val ms3 = ManagedString("你好!")
+        assertEquals("你好!", ms3.toString())
+    }
+
+    @Test
+    fun canAppend() {
+        val ms = ManagedString("Hello").append(" World!")
```

Comment:
Guess we miss append and insert tests in non-8bit locale.

Reply:
good point! I'll add them too

Reply:
done